AdyenDropIn/Views/PreApplePayView.swift

```diff
@@ -85,7 +85,7 @@ internal final class PreApplePayView: UIView, Localizable {
         
         let payButton = PKPaymentButton(paymentButtonType: .plain, paymentButtonStyle: style)
         
-        payButton.addTarget(self, action: #selector(onPayButtonTap), for: .primaryActionTriggered)
+        payButton.addTarget(self, action: #selector(onPayButtonTap), for: .touchUpInside)
```

Comment:
why not `. primaryActionTriggered` :-D ?

Reply:
i discovered that our `SubmitButton` does not work with `.primaryActionTriggered` bc it does not inherit from `UIButton`, but from `UIControl`. so when doing tests, you have to know/remember what kind of button that is to be able to trigger an event. and imho it is easier to use  `.touchUpInside` everywhere